### ISSUE TITLE
NTR: Hypergastrinemia

### ISSUE THREAD

**Issue by LCCarmody** [obophenotype/human-phenotype-ontology#4143]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Hypergastrinemia

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

An elevated amount of gastrin in the blood.

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))

Possibly: Abnormality of circulating hormone level/Abnormality of circulating gastrin level

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
Zollinger-Ellison syndrome, OMIM:126840
DUODENAL ULCER DUE TO ANTRAL G-CELL HYPERFUNCTION

**Comment by LCCarmody** [comment]
A child of this term should be Postprandial hypergastrinemia associated with OMIM:126840
DUODENAL ULCER DUE TO ANTRAL G-CELL HYPERFUNCTION

**Comment by pnrobinson** [comment]
@LCCarmody  -- these suggestions are good!
